ddl/ddl_api.go

```diff
@@ -2938,6 +2940,56 @@ func (d *ddl) ChangeColumn(ctx sessionctx.Context, ident ast.Ident, spec *ast.Al
 	return errors.Trace(err)
 }
 
+// RenameColumn renames an existing column.
+func (d *ddl) RenameColumn(ctx sessionctx.Context, ident ast.Ident, spec *ast.AlterTableSpec) error {
```

Comment:
How about re-using `getModifiableColumnJob` for this job?

Reply:
Re-use of "getModifiableColumnJob" results in "no pointer reference". Indeed, we can extract a small part of "getModifiableColumnJob" into a general function called "getSchemaAndTableInfo", about 10 lines. Is it necessary to extract it?

Reply:
So be it, never mind.